app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,66 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.JsonTask;
+import com.codesoom.assignment.application.TaskApplicationService;
+import com.codesoom.assignment.application.TaskJsonTransfer;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/tasks")
+public class TaskController {
+    TaskApplicationService taskApplicationService = new TaskApplicationService();
+    TaskJsonTransfer transfer = new TaskJsonTransfer();
+
+    @GetMapping
+    public String getAllTasks() {
+        return transfer.taskListToJson(taskApplicationService.getAllTasks()).orElseThrow();
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<?> getSpecificTask(@PathVariable Long id) {
+        return taskApplicationService.findTask(id)
+            .flatMap(
+                it -> transfer.taskToJson(it)
+            ).map(
+                it -> new ResponseEntity<>(it, HttpStatus.OK)
+            ).orElse(
+                new ResponseEntity<>(null, HttpStatus.NOT_FOUND)
+            );
+    }
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public String createTask(@RequestBody JsonTask jsonTask) {
+        Long createdTaskId = taskApplicationService.createTask(jsonTask.title);
+        return taskApplicationService.findTask(createdTaskId).flatMap(it -> transfer.taskToJson(it)).orElseThrow();
```

Comment:
이렇게 한 줄이 긴 경우에는 한 줄에 `.`이 하나씩 오도록 개행해 주면 읽는 사람에게 도움이 됩니다. 한편, method reference를 사용하면 `it`을 생략할 수 있을 것 같아요.
```suggestion
        return taskApplicationService.findTask(createdTaskId)
            .flatMap(it -> transfer.taskToJson(it))
            .orElseThrow();
```